client/signup/steps/site-type/form.jsx

```diff
@@ -10,6 +10,7 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import Card from 'components/card';
+import CountedTextarea from 'components/forms/counted-textarea';
```

Comment:
I don't think the character count is helpful in this use-case. Perhaps use a simple, single-line text input. Or consider moving this to a new step that would be shown after the customer selects "Other".

Reply:
I would like to limit text length, encourage words and avoid prose (for easy text parsing). Else, the data ceases to be very useful.

Reply:
what we did for the disconnect flow (no char limit was not a great move ):
<img width="705" alt="Screenshot 2019-03-26 at 17 17 00" src="https://user-images.githubusercontent.com/13561163/55014307-0584aa00-4feb-11e9-9ed8-44128743e367.png">